community/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "devhub-community"
+version = "0.1.0"
+authors = ["NEAR DevHub"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+near-sdk = "5.0.0-alpha.1"
+borsh = "1.0"
```

Comment:
Borsh is available via re-export from `near_sdk::borsh`, so you don't need to explicitly include it as a dependency.

Reply:
Interesting this is broken on near-sdk 5.0.0-alpha.1:
```
// I have `use near_sdk::borsh::{BorshDeserialize, BorshSerialize};`

error: proc-macro derive panicked
  --> src/lib.rs:10:10
   |
10 | #[derive(BorshDeserialize, BorshSerialize, Default)]
   |          ^^^^^^^^^^^^^^^^
   |
   = help: message: called `Result::unwrap()` on an `Err` value: Could not find `borsh` in `dependencies` or `dev-dependencies` in `/Users/bo/workspace/neardevhub-contract/community/Cargo.toml`!
```

On near-sdk 4.1.1 skipping `borsh` in Cargo.toml was working, so maybe the re-export was not in 5.0.0-alpha.1?

Reply:
@ailisp You need to be explicit like with `serde`:

```rust
#[derive(BorshDeserialize, BorshSerialize, Default)]
#[borsh(crate = "near_sdk::borsh")]
```